data/battle_scripts_1.s

```diff
@@ -550,6 +615,96 @@ BattleScript_MakeItRainDoubles:
 	jumpifword CMP_NO_COMMON_BITS, gHitMarker, HITMARKER_NO_ATTACKSTRING | HITMARKER_NO_PPDEDUCT, BattleScript_NoMoveEffect
 	goto BattleScript_MakeItRainContinuous
 
+BattleScript_EffectShedTail:
+	attackcanceler
+	attackstring
+	ppreduce
+	waitstate
+	jumpifstatus2 BS_ATTACKER, STATUS2_SUBSTITUTE, BattleScript_ButItFailed
+	jumpifbattletype BATTLE_TYPE_ARENA, BattleScript_ButItFailed
+	jumpifcantswitch SWITCH_IGNORE_ESCAPE_PREVENTION | BS_ATTACKER, BattleScript_ButItFailed
+	setsubstitute
```

Comment:
Just calling this command will cut 25% HP instead of the 50% that Shed Tail is supposed to use up.

Reply:
Bulbapedia says Shed Tail only uses 25% HP, the same as Substitute though?

Reply:
Wait, I actually misread what Bulbapedia said. Thanks for telling me, although this is actually annoying since I have to make another command or rework how substitute works like I did for belly drum......